IdokladSdk/Validation/Attributes/DateGreaterThanOrEqualThanAnotherDateAttribute.cs

```diff
@@ -8,7 +8,9 @@ namespace IdokladSdk.Validation.Attributes
 {
     public class DateGreaterThanOrEqualThanAnotherDateAttribute : ValidationAttribute
     {
-        public DateGreaterThanOrEqualThanAnotherDateAttribute(string otherPropertyName)
+        private readonly bool _allowDefualtDate;
+
+        public DateGreaterThanOrEqualThanAnotherDateAttribute(string otherPropertyName, bool allowDefaultDate = false)
```

Comment:
nemá se tohle uložit do _allowDefaultDate? A kde jsou na tuto novou logiku testy?

Reply:
Ano, opraveno. Přidal jsem nové testy.